src/worldcereal/openeo/mapping.py

```diff
@@ -124,7 +152,69 @@ def _croptype_map(
     if cropland_mask is not None:
         classes = classes.mask(cropland_mask == 0, replacement=254)
 
+    # Postprocess
+    if postprocess_parameters.enable:
+        if postprocess_parameters.save_intermediate:
+            classes = classes.save_result(format="GTiff")
+        # TODO: check how 254 value is treated during postprocessing
+        classes = _postprocess(
+            classes, postprocess_parameters, is_binary=False, lookup_table=lookup_table
+        )
+
     # Cast to uint16
     classes = compress_uint16(classes)
 
     return classes
+
+
+def _postprocess(
+    classes: DataCube,
+    postprocess_parameters: "PostprocessParameters",
```

Comment:
string still required? Thought you mentioned cyclical imports were no longer a problem?